physionet-django/console/services.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import re
+from typing import Optional
+
+from pdfminer.high_level import extract_text
+
+from user.models import Training
+
+logging.getLogger("pdfminer").setLevel(logging.WARNING)
+
+
+def _get_regex_value_from_text(text: str, regex: str) -> str:
+    result = None
+    try:
+        regex = re.compile(regex)
+        result = regex.search(text)
+        if len(result.groups()) >= 1:
+            result = regex.search(text).group(1)
```

Comment:
Taking out the first group probably deserves a comment why it's done like this, just so it's clear.